app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Команда повинна бути у форматі 'mv source destination'"
+        )
+    source = parts[1]
+    destination = parts[2]
+
+    if not os.path.exists(source):
+        raise FileNotFoundError(f"Файл {source} не існує")
+
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
+        destination = os.path.join(destination, os.path.basename(source))
```

Comment:
Code Style: It's better to use 'os.path.join' to concatenate parts of the path. It will make your code cross-platform and will use either '/' or '\' depending on the OS.